### DISCUSSION TITLE
Does validation has unique rule?

### DISCUSSION THREAD

**Discussion by Messhias** [goravel/goravel#368] [Q&A] [answered]
Hi, me here again. Sorry for the question, I'm new on Golang.

I was checking the documentation and I saw that available rules don't support for unique like Laravel.

I saw [here](https://www.goravel.dev/the-basics/validation.html#custom-validation-rules) that there's a way for us create custom validation rules. 

My question is:

How can I do something similar like:

```
"unique:users, username"
```

For example

Thanks

**Comment by hwbrzzl** [comment, marked as answer]
Yes, you need to implement a custom rule for it for now.

**Reply by Messhias** [reply]
> Yes, you need to implement a custom rule for it for now.

Nice, I'll try to do it by myself and if I make it work out I'll create the pull request.
